atomic_reactor/constants.py

```diff
@@ -14,6 +14,77 @@
 BUILD_JSON_ENV = 'BUILD_JSON'
 RESULTS_JSON = 'results.json'
 
+JSON_SCHEMA = {
```

Comment:
Can we call this plugins.schema and put it in atomic_reactor/schemas? You can retrieve it like this: https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugins/pre_reactor_config.py#L135